src/init.c

```diff
@@ -36,6 +36,54 @@ struct hyper_pod global_pod = {
 	.exec_head	=	LIST_HEAD_INIT(global_pod.exec_head),
 };
 
+struct hyper_ps_data {
+	/**
+	 * line to process, it contains information about a process
+	 * for example:
+	 *   root 5123 2 0 09:51 ? 00:00:00 [kworker/2:0]
```

Comment:
I like the example, but since this is at the top of the file, it would be good to explain that the format will vary, depending on the value of `psargs`.